app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -56,10 +63,30 @@ export default function NoteTable({
               </td>
               <td>
                 <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
+                  <a href={null}>{note.title}</a>
+                </div>
+              </td>
+              <td>
+                <div className="fixed-width-col">{note.description}</div>
+              </td>
+              <td>
+                <Badge color={note.tag.color}>{note.tag.text}</Badge>
+              </td>
+              <td>{note.createdDate}</td>
+              <td>{note.dueDate || "--"}</td>
+              <td>
+                <Avatar size={36} contact={{ name: note.contact }} />
+              </td>
+              <td>
+                <div className="flex flex-row space-x-4 items-center hover-button-wrapper">
```

Comment:
No need to provide `hover-button-wrapper`.
Please add `nui-table--actions` as className to `table` tag. This adds the hover effects on IconButtons.